starlite/config.py

```diff
@@ -177,6 +183,32 @@ def to_openapi_schema(self) -> OpenAPI:
             ),
         )
 
+    def create_openapi_schema_model(self, router: "Starlite") -> OpenAPI:
```

Comment:
Shouldn't this be typed as Router?

Reply:
I named the parameter `router` with [this](https://github.com/starlite-api/starlite/issues/241#issuecomment-1191976760) in mind. However, the method accesses `router.plugins` which doesn't exist on `Router`, only `Starlite`.

So what do you think about https://github.com/starlite-api/starlite/issues/241#issuecomment-1191976760?

If that's a non-starter, then I'll just name the parameter to this method `app` and leave the type as is. If it is a starter, eventually this would be typed `Router`.

Reply:
Then I'd rename it to app

Reply:
Are you saying that idea is a non-starter?